docs/site/content/docs/edge/getting-started-docker.md

```diff
@@ -0,0 +1,136 @@
+# Getting Started with Docker Desktop
+
+<!-- markdownlint-disable MD036 -->
+<!-- markdownlint-disable MD024 -->
+
+Docker Desktop offers a Tanzu Community Edition extension. This extension spins
+up a local cluster that you can interact with via `kubectl` and the `tanzu` CLI.
```

Comment:
```suggestion
up a local cluster that you can interact with via `kubectl` and the Tanzu CLI.
```

Reply:
this is how it's referred to throughout doc

Reply:
I wonder if `tanzu` is correct here because I'm referring to running it as a command. See `kubectl`.